src/components/strategies/overview/StrategyTable.tsx

```diff
@@ -0,0 +1,149 @@
+import { Link, useMatch } from '@tanstack/react-router';
+import { PairLogoName } from 'components/common/PairLogoName';
+import { StrategyWithFiat } from 'libs/queries';
+import { FC, useEffect, useId, useState } from 'react';
+import { StrategyStatusTag } from './strategyBlock/StrategyBlockHeader';
+import { useFiatCurrency } from 'hooks/useFiatCurrency';
+import { prettifyNumber, tokenAmount } from 'utils/helpers';
+import { StrategyGraph } from './strategyBlock/StrategyGraph';
+import { ReactComponent as DashboardIcon } from 'assets/icons/dashboard.svg';
+import {
+  ManageButtonIcon,
+  StrategyBlockManage,
+} from './strategyBlock/StrategyBlockManage';
+import styles from './StrategyContent.module.css';
+import { FiatPrice } from 'components/common/FiatPrice';
+import { Tooltip } from 'components/common/tooltip/Tooltip';
+
+interface Props {
+  strategies: StrategyWithFiat[];
+}
+export const StrategyTable: FC<Props> = ({ strategies }) => {
+  return (
+    <table className={styles.strategyTable}>
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>Token Pair</th>
+          <th>Status</th>
+          <th>Trades</th>
+          <th>Total Budget</th>
+          <th>Buy Budget</th>
+          <th>Sell Budget</th>
+          <th>Price</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        {strategies.map((strategy, index) => (
+          <StrategyRow
```

Comment:
I don't understand why you are manually managing virtualization instead of putting a virtualization container here and returning the callback to render a strategy row

Reply:
I cannot use the pattern with absolute position + translate else it breaks the table behavior.
I cannot add rows on the fly else it does layout shifting.
So I need to have empty rows, and rerender the content when it's visible in the viewport.